### ISSUE TITLE
Increasing color contrast of the Storage Bar on the Assets Page of the Profile Section

### ISSUE THREAD

**Issue by runningonsideprojects** [processing/p5.js-web-editor#3856]
### Increasing Access

This feature enhancement is targeting increasing web accessibility for visually impaired users by adhering color contrast standards to WCAG guidelines.

### Issue Description

This specific subissue aims to refactor the color contrasts of the Storage Bar on the Assets page for all themes.  

WCAG guidelines [state](https://www.allaccessible.org/blog/color-contrast-accessibility-wcag-guide-2025) that non-text color contrast should be at least 3:1. In all themes, the progress bar’s color $P5-pink and background color of the bar $gray have a contrast of just 1.1:1. This could be addressed by 1) increasing contrast between the bar color and the progress bar background, and 2) adding a border that contrasts with the page background.

Location of the requested changes on the editor: [https://editor.p5js.org/{username}/](https://editor.p5js.org/alisonkim12/sketches)assets 

### Change Requested
- [ ] Light theme
  - [ ] Change `progress-bar-background-color` to `$light`
  - [ ] Add a variable to the light theme called `progress-bar-border-color` and set it to `$middle-gray`

- [ ] Dark theme
  - [ ] Change `progress-bar-background-color` to `$darker`
  - [ ] Add a variable to the light theme called `progress-bar-border-color` and set it to `$middle-gray`

- [ ] Contrast theme
  - [ ] Change `progress-bar-background-color` to `$darker`
  - [ ] Add a variable to the light theme called `progress-bar-border-color` and set it to `$middle-light`
  - [ ] Change `progress-bar-active-color` to `$yellow`

- [ ] In `_asset-size.scss`, change the `border` from `transparent` to `progress-bar-border-color`
  - [ ] Within `@include themify()`, add a line that sets `border-color` to `getThemifyVariable(' progress-bar-border-color')`

### Visuals

Dark mode: 

<img width="512" height="119" alt="Image" src="https://github.com/user-attachments/assets/668313f3-9840-4c59-855f-e2fe5f9e0f9f" />

Light mode: 

<img width="512" height="101" alt="Image" src="https://github.com/user-attachments/assets/18bc509b-3ac9-43d7-a449-0646db650dd9" />

### Files to change
- Variables.scss ($themes),  _asset-size.scss

### PR Criteria
- [ ] Provide screenshots in ur PR of the visual changes implemented
- [ ] Provide QA steps for how reviewers can ensure your changes are made properly
- [ ] Test out by uploading assets in your local environment. You'll need to set up an AWS s3 configuration by creating an AWS s3 bucket. This is the[ main doc dedicated to outlining the steps](https://github.com/processing/p5.js-web-editor/blob/develop/contributor_docs/s3_configuration.md), which came from the[ Configuration section of the Installation doc](https://github.com/processing/p5.js-web-editor/blob/develop/contributor_docs/installation.md#s3-bucket-configuration). 

### Additional Resources:
- [Color Combinations for Accessibility](https://github.com/processing/p5.js-web-editor/wiki/Color-Combinations-for-Accessibility) – tool developed by @Izzy-Snyder that generates custom color palettes for components while adhering to WCAG guidelines and [p5.js](http://p5.js/) styling standardizations. 
- [Zeplin Style Link](https://scene.zeplin.io/project/55f746c54a02e1e50e0632c3) – the p5.js editor’s styling standardizations
- [Preparing a Pull Request](https://github.com/processing/p5.js-web-editor/blob/develop/contributor_docs/preparing_a_pull_request.md) – how-to guide on creating a Pull Request 
- [Style Guide Documentation](https://github.com/processing/p5.js-web-editor/blob/develop/contributor_docs/styling.md) (WIP), best-practices guide on implementing styling changes to the codebase.

**Comment by NishulDhakar** [comment]
Hi, I’d like to work on this issue. I’ll start implementing the requested theme updates and submit a PR soon.

**Comment by raclim** [comment]
Thanks for your interest in this issue @NishulDhakar! Just assigned it to you.

**Comment by Himanshu-0076** [comment]
Hii I am new in contribution so you can assign this issue to me.

**Comment by NishulDhakar** [comment]
Hi @Himanshu-0076 , I’ve already raised a PR for this issue. Please avoid opening additional PRs for the same issue until the maintainer reviews the existing one

**Comment by Himanshu-0076** [comment]
thanks @NishulDhakar

**Comment by Izzy-Snyder** [comment]
Hi @Himanshu-0076 - we will be adding a couple other straightforward sub-issues to this issue soon, let me know if you'd like to be assigned to them! And @NishulDhakar, @raclim may take a look tomorrow or @raclim @runningonsideprojects and me will review it on Tuesday. Thanks for your patience!
